packages/utility/src/is-experiment-enabled/is-experiment-enabled.ts

```diff
@@ -0,0 +1,7 @@
+export interface Features {
```

Comment:
We need to move shared interfaces in a models package. Feels odd now that most shared models are in `payment-integration-api` package, which causes circular dependency issues.